docs/metrics-node-exporter.md

```diff
@@ -0,0 +1,543 @@
+# Node exporter metrics
+
+Below is a list of metrics exported by the node-exporter coming from collectors that are enabled by default for Linux OS. This [link](https://github.com/prometheus/node_exporter#collectors) explains how we can enable specific collector(s) to get the metrics exported by those specific collectors.
```

Comment:
how was this list obtained?

Reply:
From the metrics endpoint of the node exporter services targetting the node exporter ds pods.